bpfd/src/server/bpf.rs

```diff
@@ -195,12 +222,32 @@ impl<'a> BpfManager<'a> {
             .collect::<Vec<_>>();
         extensions.sort_by(|(_, a), (_, b)| a.current_position.cmp(&b.current_position));
         for (id, v) in extensions.iter() {
+            let mut proceed_on = Vec::new();
```

Comment:
I think we should store this as `proceed_on: Vec<i32>` in the `struct ExetensionProgram`  (or `Metadata` if we want to include this in the sort).

You can then do:

```rust
impl ExtensionProgram {
    fn proceed_on_mask(&self) -> u32 {
      ...
    }
}
```
Instead of having this block of code here.